lib/ioki/apis/passenger_api.rb

```diff
@@ -96,6 +96,12 @@ class PassengerApi
         base_path:   [API_BASE_PATH],
         model_class: Ioki::Model::Passenger::NotificationSettings
       ),
+      Endpoints::ShowSingular.new(
+        :available_notification_settings,
+        path:        ['passenger', 'notification_settings/available'],
```

Comment:
Very nitpicky, but shouldn't it be like this? (judging from other definitions)

```suggestion
        path:        ['passenger', 'notification_settings', 'available'],
```

Reply:
Totally!